### PR TITLE
Insert 57050252 Nontakhon kerdmongkol to HelloServlet.java at line of code 32

### PR DESCRIPTION
**Pull request by nontakhon**
Please update in servlet !

### CODE REVIEW

**Comment by nontakhon** [comment]
I have changed.

File: src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java [hunk 1]
```diff
@@ -27,6 +27,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         out.println("<li>58050276 Thanathat Surakhup</li>");
         out.println("<li>58050309 Pasthiyakan Kaewpitak</li>");
         out.println("<li>58050355 Panuwich Laoudom</li>");
+	out.println("<li>57050252 Nontakhon kerdmongkol</li>");
```
Comment (champillon) [inline src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java]: please fix the alignment to align with each other line

**Comment by nontakhon** [comment]
I have changed.

**Comment by nontakhon** [comment]
have changed.

File: src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java [hunk 2]
```diff
@@ -29,6 +29,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         out.println("<li>58050355 Panuwich Laoudom</li>");
         out.println("<li>57050269 Patipon Taweechat</li>");
         out.println("<li>57050279 Piyapon Suntikan</li>");
+        out.println("<li>57050252 Nontakhon kerdmongkol</li>");
```
Comment (champillon) [inline src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java]: please pull the upstream to merge with your code.

**Pull request merged**